### ISSUE TITLE
InvalidOperationException("There is no current result set.") - reader.GetName(0)

### ISSUE THREAD

**Issue by zsakszabolcs** [bgrainger/MySqlConnector#24]
Calling reader.GetName(0) throws "InvalidOperationException". The source of the exception is in the function VerifyHasResult. The state of the reader is "State.AlreadyReadFirstRow".
 
```
		private void VerifyHasResult()
		{
			VerifyNotDisposed();
			if (m_state != State.ReadResultSetHeader && m_state != State.ReadingRows && m_state != )
				throw new InvalidOperationException("There is no current result set.");
		}
```

**Comment by bgrainger** [comment]
Should be fixed in 0.1.0-alpha07: https://www.nuget.org/packages/MySqlConnector/0.1.0-alpha07

**Comment by zsakszabolcs** [comment]
Thank you!
